### DISCUSSION TITLE
MCMT inclusion

### DISCUSSION THREAD

**Discussion by josejsarmento** [itzg/docker-minecraft-server#1377] [Ideas]
How do I add a mod such as [MCMT](https://www.curseforge.com/minecraft/mc-mods/mcmt-multithreading)? Would such a thing even work?

**Comment by itzg** [comment]
Have you read through https://github.com/itzg/docker-minecraft-server#working-with-mods-and-plugins ?

I am skeptical though of what that mod is attempting to do. If you're really wanting to improve server performance then I suggest using [Paper](https://github.com/itzg/docker-minecraft-server#running-a-paper-server).
